corehq/apps/cloudcare/static/cloudcare/js/util.js

```diff
@@ -48,7 +48,12 @@ hqDefine('cloudcare/js/util', function () {
 
     var showHTMLError = function (message, $el, autoHideTime) {
         message = message || gettext("Sorry, an error occurred while processing that request.");
-        _show(message, $el, autoHideTime, "", true);
+        var $container = _show(message, $el, autoHideTime, "", true);
+        try {
```

Comment:
I'm being "overly" cautious here because if we're wrong about that, I'd much rather it downgrade to current behavior than cause an error for the user. It may make sense after this has been up for a bit to remove some of the try / catches, after for example never seeing this downgrade the original html raw text